packages/cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "novu",
-  "version": "2.0.0-canary.1",
+  "version": "2.0.0-canary.3",
```

Comment:
Had a few errors during publish, hence the multiple version bumps 🤦‍♂️

I will work on a CI pipeline for this today.